orchestration/src/main/java/com/sap/ai/sdk/orchestration/ContentFilter.java

```diff
@@ -17,11 +18,20 @@
 public interface ContentFilter {
 
   /**
-   * A method that produces the serializable equivalent {@link FilterConfig} object from data
+   * A method that produces the serializable equivalent {@link InputFilterConfig} object from data
    * encapsulated in the {@link ContentFilter} object.
    *
-   * @return the corresponding {@code FilterConfig} object.
+   * @return the corresponding {@link InputFilterConfig} object.
    */
   @Nonnull
-  FilterConfig createConfig();
+  InputFilterConfig createInputFilterConfig();
```

Comment:
(Comment)

While I agree to the solution, it doesn't feel great that we're changing public API on convenience layer. That would be considered a breaking change normally. I do understand we only use it as workaround to expose the low-level converter method(s), but still.. 🤔

Reply:
I would have blocked the generation entirely until they reverted the spec, but I was on vacation and you accepted the breaking change. I'm still in favor of blocking changes that don't compile.